device-types/Fortinet/FG-100D.yaml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100D
+slug: fg-100d
+part_number: FG-100D
+u_height: 1.0
+is_full_depth: false
```

Comment:
Please add `weight` information if you have it

Reply:
e5019fd7ddd73918b9f8a3be553d592c36e65c4c